djangocms_link/templates/cms/plugins/link.html

```diff
@@ -1 +1 @@
-{% load cms_tags %}{% if object.child_plugin_instances %}<span class="plugin_link">{% endif %}<a href="{{ link }}"{% if target %} target="{{ target }}"{% endif %}>{% for plugin in object.child_plugin_instances %}{% render_plugin plugin %}{% empty %}{{ name }}{% endfor %}</a>{% if object.child_plugin_instances %}</span>{% endif %}
\ No newline at end of file
+{% load cms_tags %}{% if object.child_plugin_instances %}<span class="plugin_link">{% endif %}<a href="{{ link }}"{% if target %} target="{{ target }}"{% endif %}{% with attrs=object.attributes_str %}{% if attrs %} {{ attrs }}{% endif %}{% endwith %}>{% for plugin in object.child_plugin_instances %}{% render_plugin plugin %}{% empty %}{{ name }}{% endfor %}</a>{% if object.child_plugin_instances %}</span>{% endif %}
```

Comment:
don't worry about the trailing space and make it simply `` {{ object.attributes_str }}``